.github/workflows/ci.yml

```diff
@@ -23,13 +23,10 @@ jobs:
         with:
           node-version: 14.x
           cache: yarn
-      - name: Install latest NPM on Windows
+      - name: Install npm 8 on Windows
         if: matrix.os == 'windows-latest'
-        # See https://github.com/actions/virtual-environments/issues/4856#issuecomment-1043256330
-        # and https://github.com/actions/setup-node/issues/411#issuecomment-1051084491
         run: |
-          npm install -g npm@8.3.1
-          npm install -g npm@latest
+          npm install -g npm@8
```

Comment:
Not blocking on this, but since we were using 8.3.1 before, should we maybe pin the install to that version?

Reply:
Note that 8.3.1 was just used as a temporary downgrade before upgrading to the latest due to some GHA-related bug.